lib/redisHandler.js

```diff
@@ -57,17 +65,28 @@ class RedisHandler extends EventEmitter {
      * @private
      */
     _handleCommand(data) {
-        const command = data.splice(0, 1)[0];
+        let command = data.splice(0, 1)[0];
+        if (this.handleBuffers) {
+            // Command and pot. first parameter should always be Buffer
+            command = command.toString('utf-8');
+            if (data[0] && Buffer.isBuffer(data[0])) data[0] = data[0].toString('utf-8');
```

Comment:
Buffer.isBuffer returns `false` if you pass `undefined`. So removing `data[0] && ` should be fine.